docs/format.rst

```diff
@@ -260,6 +260,9 @@ A tagfield has a name, a colon, and a value: "name:value".
   are allowed.  Lower case is recommended.  Case matters ("kind:" and "Kind:
   are different tagfields).
 
+  EXCEPTION: Universal-ctags allows users to use a numerical character
+  in the name other than its initial letter.
```

Comment:
I suggest combining these two paragraphs into one, something like so:

    * The name consists of only alphanumeric characters, and must start with a letter.
      Both upper and lower case are allowed, but lower case is recommended. The name
      is case-sensitive ("kind:" and "Kind:" are different tagfields).

Reply:
...I would like to keep the original sentence that comes from Exuberant-ctags to make how universal-ctags changes the format clearly. Do you think I'm going to go wrong way?

Reply:
I did not realize the emphasis on the distinction between exuberant-ctags and universal-ctags - your reasoning makes sense to me now.

Reply:
Thank you.